README.md

```diff
@@ -1,10 +1,52 @@
-# help-to-save-stub
-
 [![Build Status](https://travis-ci.org/hmrc/help-to-save-stub.svg)](https://travis-ci.org/hmrc/help-to-save-stub) [ ![Download](https://api.bintray.com/packages/hmrc/releases/help-to-save-stub/images/download.svg) ](https://bintray.com/hmrc/releases/help-to-save-stub/_latestVersion)
 
-Run the help-to-save-stub service on port 7002, e.g.:
+## Help to Save Stub
+
+## About Help to Save
+
+Please click [here](https://github.com/hmrc/help-to-save-frontend#product-repos) for more information
+
+## How to run
 
+Run the help-to-save-stub service on port 7002, e.g.:
+```
 sbt "run 7002"
+```
+
+## How to test
+
+The unit tests can be run by running
+```
+sbt test
+```
+
+## How to deploy
+
+This microservice is deployed as per all MDTP microservices via Jenkins into a Docker slug running on a Cloud Provider.
+
+## Endpoints
+
+# POST /nsihts/createaccount
+ Given a nino of STXYZ123A this endpoint responds with a status of XYZ, XYZ being any of the possible HTTP responses.
```

Comment:
The nino doesn't need to end in `123A` - it'd be more accurate to say `Given a NINO starting with 'STxyz...`

Reply:
You're right, what I've written is a bit ambiguous.